examples/oneapi/cpp/source/knn/knn_cls_brute_force_dense_batch.cpp

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
```

Comment:
need to add here: https://github.com/oneapi-src/oneDAL/blob/master/examples/oneapi/cpp/onedal_lnx.lst and https://github.com/oneapi-src/oneDAL/blob/master/examples/oneapi/cpp/onedal_mac.lst

Reply:
done